examples/vmi-file-memory-backing.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-file-backing-memory
+  name: vmi-file-backing-memory
+spec:
+  domain:
+    devices: {}
+    memory:
```

Comment:
I think we should also showcase that we want to request higher guest memory here in order to actually get some benefit from this feature.

Reply:
Good idea!